internals/api/encryption_key.go

```diff
@@ -307,7 +307,7 @@ func (k EncryptionKeyAccountKey) Validate() error {
 	if k.Length <= 0 {
 		return ErrInvalidKeyLength
 	}
-	if k.ID.IsZero() {
+	if k.ID == uuid.UUID([uuid.Size]byte{0}) {
```

Comment:
Should this comparison be wrapped in a function?

Reply:
Yes, that would be nice!